### ISSUE TITLE
Suggestion : New Port? - In regards to previously closed 409

### ISSUE THREAD

**Issue by wesker1213** [erogenousbeef/BigReactors#433]
That one was closed, but I'd like to continue the discussion. Just on the off chance my setup is wrong, or maybe it's a bug?

I have a new reactor running with only 1 port. I have an ender io item conduit set to in/out, and it is not pulling the cyanite out. I get what you are saying when there are two ports, but in the case of having only 1, it still isn't functioning correctly. It defeats the purpose of using conduits if it doesn't work both ways as you are indicating. If I have to use buses to get it to push/pull items in and out, I may as well just use 2 ports. Thoughts?

**Comment by erogenousbeef** [comment]
I'm not entirely clear on what you mean. When you say "it is not pulling the cyanite out", do you mean that the EnderIO item conduit does not pull the cyanite out of the access port? That sounds like a bug with EnderIO.

**Comment by wesker1213** [comment]
Sorry it took a bit to get back. The only way 1 port works is to constant switch it between import, and export. Which if I'm doing that, I may as well use 2 ports. The ender io cables work if I switch it to export, but a lone port isn't enough to operate both ways, unless there is some way to toggle it between the two without opening the interface? Even with applied energistics, it is the same way.

In closing, the ports pull out, but only if it's set to export, it doesn't operate bidirectionally. 

Any guesses?

**Comment by erogenousbeef** [comment]
Ah, now I see. Yes, the inventory code only permits either the inlet or outlet inventory slots to be exposed at once.

I'll think about adding a bidirectional mode in 0.5. Until then, you'll have to work around it by using two access ports.

**Comment by wesker1213** [comment]
Fair enough. Where is version 0.5 expected, and will it be for 1.7.10? I know we're on 0.4.1 currently.

**Comment by erogenousbeef** [comment]
0.5 will be out "when it's done" and is still in fairly heavy development. It will be for 1.7.10.
